Tutorial/HighContrastSupport.md

```diff
@@ -0,0 +1,131 @@
+# Accessibility: Adding High-Contrast Mode Support
+Windows *High-Contrast* setting makes text and apps easier to see by using more distinct colors.
+Read more about [high-contrast support in Power BI](https://powerbi.microsoft.com/en-us/blog/power-bi-desktop-june-2018-feature-summary/#highContrast).
+
+Adding high-contrast support to your visual requires the following:
+1. On init: Detect whether Power BI is in high-contrast mode and if so, get current high-contrast colors.
+2. Every update: Change the way the visual renders to make it easier to see.
+
+## On Init
+The [colorPalette](./ColorPalette.md) member of `options.host` has several properties for high-contrast mode. Use these properties to determine whether high-contrast mode is active, and if yes, what colors to use.
+
+### Detect that Power BI is in high-contrast mode
+If `host.colorPalette.isHighContrast` is `true`, high-contrast mode is active and the visual should draw itself accordingly.
+
+### Get high-contrast colors
+In high-contrast mode, your visual should limit itself to the following colors:
+* **Foreground** color is used to draw any lines, icons, text and outline or fill of shapes.
+* **Background** color is used for background, and as the fill color of outlined shapes.
+* **Foreground - selected** color is used to indicate a selected or active element.
+* **Hyperlink** color is used only for hyperlink text.
+
+Note: If a secondary color is needed, foreground color may be used with some opacity (Power BI native visuals use 40% opacity). Use this sparingly to keep the visual details easy to see.
+
+You can store these values during initialization:
+
+```typescript
+    private isHighContrast: boolean;
+
+    private foregroundColor: string;
+    private backgroundColor: string;
+    private foregroundSelectedColor: string;
+    private hyperlinkColor: string;
+    //...
+
+    constructor(options: VisualConstructorOptions) {
+        this.host = options.host;
+        let colorPalette: ISandboxExtendedColorPalette = host.colorPalette;
+        //...
+        this.isHighContrast = colorPalette.isHighContrast;
+        if (this.isHighContrast) {
+            this.foregroundColor = colorPalette.foreground.value;
+            this.backgroundColor = colorPalette.background.value;
+            this.foregroundSelectedColor = colorPalette.foregroundSelected.value;
+            this.hyperlinkColor = colorPalette.hyperlink.value;
+        }
+```
+Alternatively, you can store the `host` object during initialization and access the relevant `colorPalette` properties during update.
+
+## On Update
+How exactly you draw your visual in high-contrast mode depends on your visual's specific graphic design. Usually, a somewhat different design is required to keep the important details easy to distinguish with the limited colors. 
```

Comment:
I'd rephrase this:
Each visual would typically design its own implementation...

Reply:
Done with rephrasing (a visual doesn't design its implementation, the devs [should] do that)